Comparison shopping engine/Product.cs

```diff
@@ -2,10 +2,11 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Linq;
+using Comparison_shopping_engine;
 
 namespace Comparison_shopping_engine
 {
-    public class Product
+    public class Product : IEquatable<Product>, IComparable<Product>
```

Comment:
Galėtum pakomentuoti kaip kodą pagerina kiekvienas interface?

Reply:
IEquatable dėl to kad butų galima patikrinti ar vienodi, Icomparable tai kad galima palyginti (lyginimas vyksta pagal kainas) ir beto 
![image](https://user-images.githubusercontent.com/67711718/97463095-12484d00-1948-11eb-935a-6c8504f1d7d2.png)

Reply:
Šaunu;)